preup/kickstart/application.py

```diff
@@ -210,10 +210,28 @@ def comment_kickstart_issues(self):
                 kickstart_data[index] = "#" + row
         FileHelper.write_to_file(self.kick_start_name, 'wb', kickstart_data)
 
+    def _get_all_postinstall_scripts(self):
```

Comment:
1. This method can be static as it does not handle any class data. It can be also part of utilities if implemented as the second point below suggests.
2. Why is this method named *_get* something, when the only thing you are using it for is the information whether there is/there is not at least one script in that particular folder. It seems to me boolean output would fit here. And the name like: *_contains_executable_files(dir)*